inorbit_edge/tests/test_fetch_robot_config.py

```diff
@@ -40,11 +40,11 @@
 def test_get_robot_config_from_session():
     # test required parameters only
     robot_session = RobotSession(
-        robot_id="id_123", robot_name="name_123", api_key="appkey_123"
+        robot_id="id_123", robot_name="name_123", api_key="apikey_123"
```

Comment:
this one shouldn't be "robot_apikey_123"? @leandropineda

Reply:
This is the API key you get from the developer console. It is used for authenticating against InOrbit REST API to then get the `robot_api_key`